README.md

```diff
@@ -5,10 +5,15 @@ The zigbee driver allows application to access the zigbee capabilities of the MA
 ### Pre-Requisites
 cmake, git, g++  and 0MQ
 ```
-echo "deb http://packages.matrix.one/matrix-creator/ ./" | sudo tee --append /etc/apt/sources.list;
+# Add repo and key
+curl https://apt.matrix.one/doc/apt-key.gpg | sudo apt-key add -
+echo "deb https://apt.matrix.one/raspbian $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/matrixlabs.list
+
+# Update packages and install
 sudo apt-get update;
 sudo apt-get upgrade;
 sudo apt-get install cmake g++ git;
+sudo apt-get install --yes libmatrixio-malos-dev
```

Comment:
Here are the correct dependencies: `apt-get install --yes libmatrixio-malos-dev libmatrixio-protos-dev`  if you have any doubts with the dependencies and compiling rules, you could check the .drone.yml